### PR TITLE
fix: add himanshu1221

### LINKED ISSUE #1698: [PROFILE ADDITION] <Himanshu Chhatwal>
### Profile screenshot

Screenshot from 2023-02-02 22-21-48

### PR DESCRIPTION
**Pull request by himanshu1221**
resolves #1698 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

### CODE REVIEW

File: profiles/himanshu1221.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://avatars.githubusercontent.com/u/32031706?v=4",
```
Comment (Panquesito7) [inline profiles/himanshu1221.json:2]: The image must be the image link provided in the issue you created (#1698).
```suggestion
  "image": "https://user-images.githubusercontent.com/32031706/216389505-bff1996b-1553-4867-bc8e-05eba9aabb6a.png",
```

Comment (himanshu1221) [reply profiles/himanshu1221.json:2]: okay :+1:

**Comment by himanshu1221** [comment]
@Panquesito7 is it okay now ?

**Review by lewisushindi — APPROVED** [review]

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻 ⚡

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**